Provider/Pool.php

```diff
@@ -28,11 +28,18 @@ class Pool
     protected $contexts = array();
 
     /**
+     * @deprecated Deprecated since version 3.x and will be removed in 4.0. Use $downloadStrategies instead.
```

Comment:
@core23 is right. 3.x should be put. It will be replaced just before next release.